gradle-plugins/src/main/kotlin/com/arcgismaps/CopySampleFilesTask.kt

```diff
@@ -42,6 +48,58 @@ class CopySampleFilesTask : Plugin<Project> {
 
             // Place in the assets codeFiles folder of the app.
             into(outputDir)
+
+            // After the copy finishes, run JSON generation + cleanup
+            doLast {
+                // Create a JSON file that maps each sample folder’s files to their contents
+                createJsonAssetFile(outputDir)
+
+                // Delete everything but the JSON file itself
+                outputDir.listFiles()?.forEach { file ->
+                    if (file.isDirectory || file.name != "samples.json") {
+                        file.deleteRecursively()
```

Comment:
So this should handle the transition for users on the current sample viewer--their sample metadata, code files, READMEs should all get deleted.

Reply:
Users (downloading from PlayStore), will not need to run the `CopySampleFilesTask`, but instead the SV will be pre-loaded with the `samples.json` in the assets folder. It would be when users download a new app-version  `Room` would detect a change in the [versionCode](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/model/room/AppDatabase.kt#L30), which should make a new instance of the DB with the JSON file changes. Which will be responsible for handling the transition for users on the current sample viewer.

Reply:
Yeah I mean for users who have a local copy from github, which I think is probably still the majority use case of samples.

Reply:
Tested switching `v.next` and this branch while only running a sync before a build and the updated cleanup task was able to handle the deletion process.